packages/ui/src/services/xpath/2.0/xpath-2.0-parser.ts

```diff
@@ -96,7 +97,7 @@ const AncestorOrSelf = createToken({ name: 'AncestorOrSelf', pattern: /ancestor-
 const Ancestor = createToken({ name: 'Ancestor', pattern: /ancestor/, longer_alt: NCName });
 const PrecedingSibling = createToken({ name: 'PrecedingSibling', pattern: /preceding-sibling/, longer_alt: NCName });
 const Preceding = createToken({ name: 'Preceding', pattern: /preceding/, longer_alt: NCName });
-const ContextItemExpr = createToken({ name: 'ContextItemExpr', pattern: /\./ });
+const ContextItemExpr = createToken({ name: 'ContextItemExpr', pattern: /\./, longer_alt: DecimalLiteral });
```

Comment:
The point is this, `longer_alt` has to be specified so that the parser can fallback to take it as a decimal when a dot appears with numbers